src/Console/ConfigurationResolver.php

```diff
@@ -513,6 +509,22 @@ private function computeConfigFiles()
     }
 
     /**
+     * @return FixerFactory
+     */
+    private function createFixerFactory()
+    {
+        static $fixerFactory = null;
```

Comment:
now it will be cached across different instances of `ConfigurationResolver`. very problematic for tests and using `PHP-CS-Fixer` as library. static is global, and then you base it on config, which may be different for different `ConfigurationResolver` instances